shaal/DrupalPod
# Feature: replace .vscode directory with a .code-workspace file

# The Problem/Issue/Bug

The .vscode directory stores files that can be used to configure the editor's settings, debugger, et cetera, but the same thing and more can be accomplished in a .code-workspace file. Additionally, the code-workspace file allows users to create "multi-root workspaces" which can make navigating code easier when applications are spread across large numbers of files.

## How this PR Solves The Problem

This PR moves the existing launch.json and settings.json files into the code workspace and sets three workspace folders:
- DrupalPod (/workspace/DrupalPod/)
- Drupal Core (/workspace/DrupalPod/repos/drupal/)
- Web Root (/workspace/DrupalPod/web)

Setting the folders up this way let us remove the distractions of other files/folders when we want to focus on work in the webroot or drupal git repo.

## Manual Testing Instructions

Open the deploy preview, which should automatically set the editor's workspace to use the code-workspace file.

## Related Issue Link(s)

None

## Release/Deployment notes
